testing/btest/spicy/udp-wrong-port.zeek

```diff
@@ -0,0 +1,40 @@
+# @TEST-REQUIRES: have-spicy
```

Comment:
Looks good to me, just nits, feel free to ignore.

We could instead parse
```
data: bytes &eod. { assert False: "cannot be reached"; }`
```
That would make the Zeek script unnecessary and data forwarding unnecessary.

We could also include the output of `HILTI_DEBUG=zeek zeek ...` in the baseline (possible reduced). This contains spicy-plugin logging for the registration.